lib/commands/migrate.js

```diff
@@ -14,23 +14,18 @@ const WebpackOptionsValidationError = require("webpack")
 const runPrettier = require("../utils/run-prettier");
 
 /**
-*
-* Runs migration on a given configuration using AST's and promises
-* to sequentially transform a configuration file.
-*
-* @param {String} currentConfigPath - Location of the configuration to be migrated
-* @param {String} outputConfigPath - Location to where the configuration should be written
-* @param {Object} options - Any additional options regarding code style of the written configuration
+ *
+ * Runs migration on a given configuration using AST's and promises
+ * to sequentially transform a configuration file.
+ *
+ * @param	{String}	configPath - Location of the configuration to be migrated/should be written
+ * @param	{Object}	options - Any additional options regarding code style of the written configuration
+ *
+ * @returns	{Promise}	Runs the migration using a promise that will throw any errors during each transform
+ * or output if the user decides to abort the migration
+ */
 
-* @returns {Promise} Runs the migration using a promise that will throw any errors during each transform
-* or output if the user decides to abort the migration
-*/
-
-module.exports = function migrate(
-	currentConfigPath,
-	outputConfigPath,
-	options
-) {
+module.exports = function migrate(configPath, options) {
```

Comment:
I don't get this change. Why a parameter has been removed?

Reply:
https://github.com/webpack/webpack-cli/blob/b6a30ad94a100440b040b3533d90960582bc42b0/lib/index.js#L31

We were using the same variable. It was redundant to have 2 params for a value coming from a single source.

Reply:
So maybe the bug is actually there when we call migrate.js, not here. I don't think we should change the signature of the function, we are basically changing the semantic and functionality of the function, I think.

Reply:
I agree but AFAIK we aren't using migrate module at any other places in the code. This should fix the new params signature.